client/src/App.jsx

```diff
@@ -1,5 +1,31 @@
+import { useState, useEffect } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from './layouts/header/Header';
+import { setThemeToLocalStorage, getThemeFromlocalStorage } from './services/themeService';
+
 function App() {
-  return <div>Hello world!</div>;
+
+  const[theme, setTheme] = useState('light');
+
+  const themeToggle = () => {
+    const newTheme = theme == 'light' ? 'dark': 'light';
```

Comment:
TRy to never ever use magic strings like these ones. Create respective constants